src/components/v5/common/ActionSidebar/partials/ExpenditureBadge/consts.ts

```diff
@@ -1,11 +1,20 @@
-import { ExpenditureStatus } from '~gql';
+import { tw } from '~utils/css/index.ts';
+
+import { ExpenditureActionStatus } from './types.ts';
 
 export const EXPENDITURE_STATE_TO_CLASSNAME_MAP: Record<
-  ExpenditureStatus,
+  ExpenditureActionStatus,
```

Comment:
It's nit-picky but could you check the naming consistency (now it's using status, step and state to describe the same thing) to avoid confusion? I like your idea of "expenditure action status"